docs/getting-started/install.rst

```diff
@@ -16,6 +16,7 @@ Wagtail SEO also requires the Wagtail Site Settings app and context processor.
 
     INSTALLED_APPS = [
         ...
+        "wagtail.contrib.settings",
```

Comment:
Minor nit pick... could you move this below `wagtailseo,`. So that the wagtail apps are grouped together.